package.json

```diff
@@ -15,12 +15,15 @@
   },
   "devDependencies": {
     "benchmark": "^2.1.4",
-    "fastify": "^4.3.0",
+    "fastify": "^4.26.2",
     "snazzy": "^9.0.0",
-    "standard": "^17.0.0",
-    "tap": "^16.3.0",
+    "standard": "^17.1.0",
+    "tap": "^18.7.2",
     "tsd": "^0.31.0"
   },
+  "resolutions": {
+    "@types/node": "^20.12.7"
+  },
```

Comment:
Is this necessary?

Reply:
I added that because CI was failing with Yarn: https://github.com/fastify/one-line-logger/actions/runs/8755573058/job/24070941997 
I can try to locally re-run the command, and see if something has changed

Reply:
Just to give you more info, when running `yarn` to install deps, we got some warnings:
```
warning "tap > @tapjs/asserts > tcompare > react-element-to-jsx-string@15.0.0" has unmet peer dependency "react@^0.14.8 || ^15.0.1 || ^16.0.0 || ^17.0.1 || ^18.0.0".
warning "tap > @tapjs/asserts > tcompare > react-element-to-jsx-string@15.0.0" has unmet peer dependency "react-dom@^0.14.8 || ^15.0.1 || ^16.0.0 || ^17.0.1 || ^18.0.0".
warning "tap > @tapjs/test > @isaacs/ts-node-temp-fork-for-pr-2009@10.9.5" has unmet peer dependency "@types/node@*".
```

Please do note the last one. I dig a little and I can see here (line 153 https://www.npmjs.com/package/@isaacs/ts-node-temp-fork-for-pr-2009?activeTab=code) that `@isaacs/ts-node-temp-fork-for-pr-2009` has a peer dependency on `@types/node`.

`npm` should automatically intall `peerDependencies` from v7 (https://github.com/npm/rfcs/blob/main/implemented/0025-install-peer-deps.md), but from what I can gather on Google, it seems `yarn` does NOT install those, so.. 
I'm wondering then why the other two unmet peer dependency does not brake the `yarn run test`, but maybe that's because when running that command, we execute code of `tap` package that does not rely on `react` nor `react-dom`